modules/openapi-generator/src/main/resources/dart2/class.mustache

```diff
@@ -108,6 +108,8 @@ class {{classname}} {
   {{/isDate}}
   {{/isDateTime}}
   {{/vars}}
+    if (false{{#vars}}{{#required}} || {{name}} == null{{/required}}{{/vars}})
+      throw ArgumentError("$json has not all required fields for {{classname}}");
```

Comment:
Suggestion: what about checking these required fields one by one (to avoid hardcoded `false` at the start) and including the missing required fields in the error message?